proposed/2021/CentralPackageManagementTransitive.md

```diff
@@ -0,0 +1,201 @@
+# Specifying Versions for Transitive Dependencies
+Central Package Version Management (CPVM) is a feature set in NuGet that allows you to manage package versions in one place.  Consider the following packages are available:
+
+| Id | Version | Dependencies |
+|-----:|:---------|-----|
+| PackageA | 1.0.0 | PackageC >= 3.0.0 |
+| PackageB | 2.0.0 | PackageC >= 4.0.0 |
+| PackageC | 3.0.0 |  |
+| PackageC | 4.0.0 |  |
+| PackageC | 5.0.0 |  |
+
+## Referencing packages without CPVM
+This section explains how to specify a different version of a transitive dependency when using the standard mechanism for referencing packages and specifying versions.
+
+**`ClassLibrary1.csproj`**
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="PackageA" Version="1.0.0" />
+    <PackageReference Include="PackageB" Version="2.0.0" />
+  </ItemGroup>
+</Project>
+```
+
+`PackageA` and `PackageB` both transitively depend on `PackageC` but different versions, `3.0.0` and `4.0.0` respectively.  NuGet will unify this to the minimum version that satisifies the requirements which is **`4.0.0`**.
+
+During Pack, the resulting package will have the top-level dependencies on `PackageA` and `PackageB` and the transitive dependencies of `PackageC` will be resolved by the consumer:
+
+**`ClassLibrary1.nuspec`**
+```xml
+<dependencies>
+  <group targetFramework="net5.0">
+    <dependency id="PackageA" version="1.0.0" exclude="Build,Analyzers" />
+    <dependency id="PackageB" version="2.0.0" exclude="Build,Analyzers" />
+  </group>
+</dependencies>
+```
+Since version `4.0.0` of `PackageC` was used in this scenario, it is okay for the generated package to transitively depend on `4.0.0` since consumers of this library will end up using hte same version of `PackageC` that was used by this package.
```

Comment:
```suggestion
Since version `4.0.0` of `PackageC` was used in this scenario, it is okay for the generated package to transitively depend on `4.0.0` since consumers of this library will end up using the same version of `PackageC` that was used by this package.
```